VideoLocker/src/main/java/org/edx/mobile/services/EdxCookieManager.java

```diff
@@ -0,0 +1,111 @@
+package org.edx.mobile.services;
+
+import android.content.Context;
+import android.os.Build;
+import android.webkit.ValueCallback;
+
+import org.apache.http.cookie.Cookie;
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.event.SessionIdRefreshEvent;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.module.prefs.PrefManager;
+import org.edx.mobile.task.GetSessesionExchangeCookieTask;
+
+import java.io.File;
+import java.util.List;
+
+import de.greenrobot.event.EventBus;
+
+/**
+ *  A central place for course data model transformation
+ */
+public class EdxCookieManager {
+
+    protected final Logger logger = new Logger(getClass().getName());
+
+    private static EdxCookieManager instance;
+
+    private GetSessesionExchangeCookieTask task;
+
+    public static synchronized EdxCookieManager getSharedInstance(){
+        if ( instance == null )
+            instance = new EdxCookieManager();
+        return instance;
+    }
+
+    public void clearWebWiewCookie(Context context){
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            android.webkit.CookieManager.getInstance().removeAllCookies(new ValueCallback() {
```

Comment:
Can we not clear the cookies at all before lollipop? If a user logs out, we need to remove their authentication tokens.